capture-test-execution-system-properties/gradle-test-execution-system-properties.gradle

```diff
@@ -1,10 +1,12 @@
+import java.nio.charset.StandardCharsets
 import java.security.MessageDigest
 
 /**
  * This Gradle script captures the system properties available to each Test task, hashes the properties' values,
  * and adds these as custom values.
+ * This should be applied to the root project since it configures all projects:
+ * <code> apply from: file('gradle-test-execution-system-properties.gradle') </code>
  */
```

Comment:
Can you please check if there are other capture* scripts where we should add the same note?